_studio/mfx_lib/fei/h264_la/mfx_h264_la.cpp

```diff
@@ -614,7 +614,7 @@ mfxStatus VideoENC_LA::InsertTaskWithReordenig(sLAInputTask &newTask, bool bEndO
     { 
         m_numLastB = 0;
         m_miniGop.resize(0);
-        sLAInputTask finished_task = {0};
+        sLAInputTask finished_task =sLAInputTask();
```

Comment:
Here you use default initialization of POD struct to create temporary zero-initialized sLAInputTask object and then copy it. You don't need temporary object, just initialize the finished_task object in place:
```
sLAInputTask finished_task();
```